slides/src/content/original/routing.md

```diff
@@ -0,0 +1,297 @@
+# Routing
+
+---
+
+## Why Routing?
+
+- Routing allows the user to navigate to different parts of the application.
+- It keeps your application linkable and bookmarkable and allow users to share links with others.
+
+Routing allows you to:
+
+- Maintain the state of the application.
+- Create modular applications (every module define its own routes).
+- Implement authorization (certain roles have access to certain URLs).
+
+Note:
+
+- Routing is optional, we can build an application that never changes the URL.
+
+---
+
+## Configuring Routes (1/3) - Base URL Tag
+
+The Base URL tag must be set within the `<head>` tag of index.html:
+
+```html
+ <base href="/">
+```
+
+This tells the router how to compose navigation URLs.
+
+Notes:
+
+In the demos we use a script tag to set the base tag. In a real application it must be set as above.
```

Comment:
> In the demos we use a script tag to set the base tag. In a real application it must be set as above.

This is confusing. Isn't the base href always required unless you set it programmatically?